### ISSUE TITLE
name 'inner' is not defined

### ISSUE THREAD

**Issue by chalesguo** [JeanKossaifi/tensorly-notebooks#2]
in tensor_regression_layer_pytorch, i got a error:

<ipython-input-27-054dbfcec6ba> in forward(self, x)
     28     def forward(self, x):
     29         regression_weights = tl.tucker_to_tensor(self.core, self.factors)
---> 30         return inner(x, regression_weights, n_modes=tl.ndim(x)-1) + self.bias
     31 
     32     def penalty(self, order=2):

NameError: name 'inner' is not defined

**Comment by JeanKossaifi** [comment]
If you update TensorLy to the latest version, this should get rid of the problem. 

With pip:
```bash
pip install -U tensorly
```

The issue is that the function was not imported in `tensorly/tenalg/__init__.py`, this was fixed by https://github.com/tensorly/tensorly/commit/9f6130991a6046aba1831a6295660802bab2b515
If you have cloned the repository you can also fix the issue either by editing directly that file the following: 
```bash
from .generalised_inner_product import inner
```

**Comment by chalesguo** [comment]
i change the file of  tensor_regression_layer_pytorch, so that is OK.

def forward(self, x):
        regression_weights = tl.tucker_to_tensor(self.core, self.factors)
        return **tl.tenalg.inner**(x, regression_weights, n_modes=tl.ndim(x)-1) + self.bias

**Comment by JeanKossaifi** [comment]
Thanks, I had missed that one!
Wanna do a quick Pull-Request to add your changes to the repo?

**Comment by JeanKossaifi** [comment]
Fixed in 43c69526e43305ee28b1e860314eb9b9bf4511fa
